pyanaconda/modules/payloads/payload/live_image/installation.py

```diff
@@ -27,37 +27,50 @@
 class InstallFromTarTask(Task):
     """Task to install the payload from tarball."""
 
-    def __init__(self, tarfile_path, dest_path):
+    def __init__(self, sysroot, tarfile):
+        """Create a new task.
```

Comment:
Very tangential, but docstgrings like this are useless. It's init of a thing called Task, I know I'm creating a task. Too bad we have these as copypasta on all tasks, IIRC... At least that means it's a collective blame thing.

On a second thought, maybe the docstring from run() is the one with the real information, in most cases.

Reply:
I use the docsting of the `__init__` method to document arguments of the task. The main description should be documented in the docstring of the class and the implementation and results in the `run` method. There needs to be something written on the first line of each docstring.

I don't understand how would you like to improve it. `Create a new task` is as useless as `Create a new task for installation of the payload from tarball`, because it's init of a thing documented as `Task to install the payload from tarball`.

Reply:
Hm, true.